src/components/Faq/FaqAccordion.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { MinusIcon, AddIcon } from '@chakra-ui/icons'
+import {
+  Accordion,
+  AccordionItem,
+  Text,
+  AccordionButton,
+  Box,
+  AccordionPanel,
+  Flex,
+  useColorModeValue,
+  Icon,
+} from '@chakra-ui/react'
+
+const SingleAccordion = ({
+  header,
+  body,
+}: {
+  header: string
+  body: string
+}) => {
+  const accordionBg = useColorModeValue('#ffffff', '#2D3748')
+  const accordionShadow = useColorModeValue('0px 0px 20px 11px #80808012', '')
+  return (
+    <Flex direction="column">
+      <Accordion defaultIndex={[1]} allowMultiple my={{ base: 5 }}>
+        <AccordionItem
+          borderRadius="5"
+          py={{ base: 5 }}
+          bg={accordionBg}
+          border="none"
+          boxShadow={accordionShadow}
+        >
+          {({ isExpanded }) => {
+            return (
+              <>
+                <Text
+                  fontWeight="bold"
+                  fontSize={{ base: 'sm', md: 'md', lg: 'xl' }}
+                >
+                  <AccordionButton
+                    _hover={{ bg: 'none' }}
+                    _focus={{ outline: 'none' }}
+                  >
+                    <Box flex="1" textAlign="left">
+                      <Text
+                        fontWeight="bold"
+                        fontSize={{ base: 'sm', md: 'md', lg: 'md' }}
+                      >
+                        {header}
+                      </Text>
+                    </Box>
+                    {/* <AccordionIcon /> */}
+                    {isExpanded ? (
+                      <MinusIcon fontSize="12px" />
+                    ) : (
+                      <AddIcon fontSize="12px" />
+                    )}
+                  </AccordionButton>
+                </Text>
+                <AccordionPanel pb={4}>
+                  <Text
+                    fontSize={{ base: 'sm', md: 'sm', lg: 'sm' }}
+                    lineHeight={{ base: '2' }}
+                  >
+                    {body}
+                  </Text>
+                </AccordionPanel>
+              </>
+            )
+          }}
+        </AccordionItem>
+      </Accordion>
+    </Flex>
+  )
+}
+const FaqAccordion = ({
+  faqData,
+  heading,
+  icon,
+}: {
+  faqData: Array<any>
```

Comment:
Avoid the `any` type please, it defeats the purpose of typescript.
faqData is an array of objects, so give it that type.
```suggestion
  faqData: Record<string, string>[]
```

Find out more [here](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type) to know exactly how that type works, and if you might need to tweak it.